README.md

```diff
@@ -70,12 +70,7 @@ spec:
 
 ## User guide
 
-
-* [Application Workload Author's Guide](docs/application-author.md)
-* [Backing Service Provider Best Practices Guide](docs/BackingServiceBestPractices.md)
-
-
-
+* [Service Binding Operator User Guide](docs/SBO_User_Guide.md)
```

Comment:
We have now just single bullet point in `User Guide` section - How about that we provide the link in the title itself:

```
## [User Guide](docs/SBO_User_Guide.md)
```